pyproject.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
     "mypy",
     "types-PyYAML",
     "types-requests",
+    "black"
```

Comment:
We use "ruff fmt" and phasing out black

Reply:
`make lint` still required black, but for now i just skipped from running it.